awx/ui_next/src/screens/Template/WorkflowJobTemplate.jsx

```diff
@@ -107,22 +107,17 @@ class WorkflowJobTemplate extends Component {
       tab.id = n;
     });
 
-    let cardHeader = hasContentLoading ? null : (
-      <TabbedCardHeader>
-        <RoutedTabs tabsArray={tabsArray} />
-        <CardActions>
-          <CardCloseButton linkTo="/templates" />
-        </CardActions>
-      </TabbedCardHeader>
-    );
-
-    if (location.pathname.endsWith('edit')) {
-      cardHeader = null;
-    }
     if (hasContentLoading) {
-      return <ContentLoading />;
+      return (
```

Comment:
If `hasContentLoading` is true, this function returns early. There is no reason or need for the code to ever reference `hasContentLoading` past this point.